lib/PuppeteerSharp/ConnectOptions.cs

```diff
@@ -13,12 +13,7 @@ public class ConnectOptions : IBrowserOptions, IConnectionOptions
         /// Whether to ignore HTTPS errors during navigation. Defaults to false.
         /// </summary>
         public bool IgnoreHTTPSErrors { get; set; }
-
-        /// <summary>
-        /// If set to true, sets Headless = false, otherwise, enables automation.
-        /// </summary>
-        public bool AppMode { get; set; }
```

Comment:
seems weird this isnt used?

Reply:
We dropped that here https://github.com/hardkoded/puppeteer-sharp/pull/529/files .

Can we mark this as obsolete?

Reply:
fixed